spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  describe 'initialize' do
+    it 'creates a new facility factory' do
+      factory = FacilityFactory.new
+      expect(factory).to be_an_instance_of(FacilityFactory)
+    end
+  end
+
+  describe '#create_facilities(state_abbv)' do
+    it 'creates DMVs specific to the specified state' do
+      factory = FacilityFactory.new
+      facilities = factory.create_facilities("CO")
+      facilities.each do |facility|
+      expect(facility).to be_an_instance_of(Facility)
+      expect(facility.address).to include("CO")
+      expect(factory.create_facilities("HI")).to eq("sorry, we don't yet serve HI")
```

Comment:
I think this line should be outside of the `do`/`end` block, the assertion doesn't need to run for every `facility` from line 14.